### DISCUSSION TITLE
Support hexagon cells in GridMap

### DISCUSSION THREAD

**Discussion by Wasabi-Cheetah** [Redot-Engine/redot-proposals#41] [Ideas]
This was a previously asked for Pull Request in the Godot repo that was first developed by octetdev2:
https://github.com/godotengine/godot-proposals/issues/4337

Due to octetdev2's lack of activity, it's been maintained, refined and polished by BSChad:
https://github.com/godotengine/godot/pull/85890

It seems stable and well refined at the moment.

Many people have requested this feature for creating turn based, DnD-like and other tile style games that would otherwise not work on the basic grid.

**Comment by Arctis-Fireblight** [comment]
Following up on these old proposals.
Looks like this is still stuck in Limbo: https://github.com/godotengine/godot/pull/85890

I would be willing to greenlight bringing this into Redot LTS if you are willing to resubmit your PR.
